<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>账  号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwd_line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登  录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signin_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注  册</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logo_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>203</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowLogo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密  码：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
